### ISSUE TITLE
Adding negative sighting in addition to positive sighting

### ISSUE THREAD

**Issue by adulau** [MISP/MISP#1522]
Adding negative sighting in addition to positive sighting. In addition to the standard model of sighting described in STIX 1.x:

```
Observed Data is not an intelligence assertion,
it is simply information: this file was seen, without any context for what it means.
```

MISP should provide a way to do negative sighting where sighting has a meaning on his own. @iglocska proposed a flag in the API to update a parallel counting system for the sighting.

**Comment by iglocska** [comment]
Additionally: Add a comment with the sighting (such as where / how was it seen etc)

**Comment by adulau** [comment]
Proposal for improvement described here: https://github.com/MISP/MISP/wiki/EnhancedSighting

**Comment by adulau** [comment]
Implemented in 2.4.66
